### PR TITLE
Webui tools_only mode

### PR DESCRIPTION
**Pull request by hascar**
Management of tools_only mode the webui

The tests are green here: https://jenkins.vpn.openquake.org/job/zdevel_oq-platform-tools/6/

### CODE REVIEW

**Review by vot4anto — APPROVED** [review]
LGTM

File: openquake/server/local_settings.py.tools.tmpl [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# This feature requires:
```
Comment (ptormene) [inline openquake/server/local_settings.py.tools.tmpl:1]: Please rename this file to local_settings.py.tools (without tmpl) to make it consistent with other existing templates

**Review by ptormene — APPROVED** [review]
If it is ok to have LOCKDOWN=False, LGTM

File: openquake/server/local_settings.py.tools [hunk 2]
```diff
@@ -0,0 +1,41 @@
+# This feature requires:
+#  - https://github.com/FirefighterBlu3/python-pam
+#  - https://github.com/cnobile2012/django-pam
+
+from openquake.server.settings import INSTALLED_APPS, AUTHENTICATION_BACKENDS
+
+# Enable authentication
+LOCKDOWN = False
```
[the remark this answers is not in the document]
Comment (vot4anto) [reply openquake/server/local_settings.py.tools:8]: Yes, the tools_only mode does not require authentication

File: openquake/server/local_settings.py.tools.tmpl (hunk 1 above)
Comment (hascar) [reply openquake/server/local_settings.py.tools.tmpl]: Solved with this commit: https://github.com/gem/oq-engine/pull/9531/commits/64022863b096c32cc6a1ebaf786b97bd98e9f545

File: openquake/server/local_settings.py.tools [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+# WEBUI config uncomment and set properly if needed
+# WEBUIURL = 'http://localhost:8800/'
+
+# turn on USE_X_FORWARDED_HOST to expose the webui via a proxy server
+# USE_X_FORWARDED_HOST = True
+
+# enable WEBUI_PATHPREFIX to place webui pages below a specific path
+# WEBUI_PATHPREFIX='/path/prefix'
+
+WEBUI_ACCESS_LOG_DIR = '/var/log/oq-engine'
```
Comment (ptormene) [inline openquake/server/local_settings.py.tools:20]: Without authentication, I suppose this is useless

**Pull request merged**